x-pack/legacy/plugins/ml/public/application/services/results_service/result_service_rx.ts

```diff
@@ -532,3 +532,11 @@ export function getScheduledEventsByBucket(
       })
     );
 }
+
+export function fetchPartitionFieldsValues(
+  criteriaFields: Array<{ fieldName: string; fieldValue: any }>,
```

Comment:
As above, for consistency with the other functions in here, I think job ID should be passed as a separate parameter, rather than including it in criteriaFields.